groups/sig-cluster-lifecycle/groups.yaml

```
@@ -207,11 +207,10 @@ groups:
     settings:
       ReconcileMembers: "true"
     members:
-      - davanum@gmail.com
-      - detiber@gmail.com
-      - goldsteina@vmware.com
-      - prignanov@vmware.com
+      - buringerst@vmware.com
```

```suggestion
      - sbueringer@gmail.com
```
I use my private mail for permissions and the VMware one only for CI alerts